src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/dto/FortisRecords.scala

```diff
@@ -99,6 +99,21 @@ case class PopularTopicAggregate(
                                   topic: String
                                 ) extends AggregationRecordTile with Serializable
 
+case class ConjunctiveTopicAggregate(
+                                      conjunctivetopic: String,
```

Comment:
Hey can you fix the indentation in this file to follow our convention of 2 space indents while you're here? :)